test/model.test.ts

```diff
@@ -225,7 +225,7 @@ function customFunctionWrapper(...args: any[]): boolean {
 test('TestKeyMatchCustomModel', () => {
   const e = new Enforcer('examples/keymatch_custom_model.conf', 'examples/keymatch2_policy.csv');
 
-  // e.addFunction('keyMatchCustom', customFunctionWrapper);
+  e.addFunction('keyMatchCustom', customFunctionWrapper);
```

Comment:
I have fixed this line in: https://github.com/casbin/node-casbin/commit/01c735be548f4c0cd18ab1bb4aa2b00bb7d5f5f5. Please remove it.